components/permissions/permission_lifetime_utils.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
filename should match the class/struct name permission_lifetime_option.h

Reply:
also if this is strictly for the UI it doesn't seem necessary to create a component for it

Reply:
although with https://github.com/brave/brave-core/pull/7982/files#diff-c9a3888745c070d869dec7c6aa211435f4d4bcecb3b4c5ef4a9413dad8de08a3 it could make sense to keep the component here and add the features here

Reply:
Yep, it will be used everywhere. The PR with main logic will be landed after this UI work.

Reply:
extracted ```PermissionLifetimeOptions``` to dedicated file, kept ```utils```.